domains/portal.maragi.is-cool.dev.json

```diff
@@ -0,0 +1,14 @@
+{
+    "domain": "is-cool.dev",
+    "subdomain": "portal.maragi",
+
+    "owner": {
+        "email": "maragidevelopments@gmail.com"
+    },
+
+    "record": {
+        "A": ["185.31.40.24"]
```

Comment:
404 site, please provide a preview of the site.

Reply:
it would just be a login screen but ok

Reply:
https://maragi.alwaysdata.net/